src/codegen/metadata/script_info.py

```diff
@@ -28,5 +28,123 @@
             "relativeOutputPath": "_task_modules\\channels\\do_channel.py", 
             "templateFile": "_task_modules\\channels\\do_channel.py.mako",
         },
+    ],
+    "modulesToCopy":[
```

Comment:
instead can we copy all files and skip files that are already present?

Reply:
updated the code to use mako files